packages/suite/src/views/wallet/trading/common/TradingForm/TradingFormOffer.tsx

```diff
@@ -145,6 +146,18 @@ export const TradingFormOffer = () => {
             return;
         }
 
+        if (isTradingSellContext(context)) {
+            const provider = quote.exchange
+                ? context.sellInfo?.providerInfos[quote.exchange]
+                : undefined;
+
+            if (provider?.flow === 'BANK_ACCOUNT') {
+                dispatch(tradingSellActions.setFormStep('BANK_ACCOUNT'));
+            } else {
+                dispatch(tradingSellActions.setFormStep('SEND_TRANSACTION'));
+            }
+        }
+
         dispatch(tradingExchangeActions.setFormStep('RECEIVING_ADDRESS'));
```

Comment:
this should be probably moved into `isTradingExchangeContext(context)` condition